effects/CMakeLists.txt

```diff
@@ -18,18 +18,25 @@ else (APPLE)
       set(INCS "")
 endif (APPLE)
 
-QT5_ADD_RESOURCES (qrc_effects_files
-     zita1/zita.qrc
-     )
+if (Qt5Widgets_VERSION VERSION_GREATER_EQUAL "5.11.0")
+      find_package(Qt5QuickCompiler)
+      qtquick_compiler_add_resources (qrc_effects_files
+            zita1/zita.qrc
+            )
+else (Qt5Widgets_VERSION VERSION_GREATER_EQUAL "5.11.0")
+      QT5_ADD_RESOURCES (qrc_effects_files
+            zita1/zita.qrc
+            )
+endif (Qt5Widgets_VERSION VERSION_GREATER_EQUAL "5.11.0")
 
 QT5_WRAP_UI (ui_headers
       compressor/compressor_gui.ui
       )
 
 if (NOT MSVC)
-   set(_all_h_file "${PROJECT_BINARY_DIR}/all.h")
+      set(_all_h_file "${PROJECT_BINARY_DIR}/all.h")
 else (NOT MSVC)
-   set(_all_h_file "${PROJECT_SOURCE_DIR}/all.h")
+      set(_all_h_file "${PROJECT_SOURCE_DIR}/all.h")
 endif (NOT MSVC)
```

Comment:
I have long wanted to know, maybe you know, why for not MSVC `all.h` is taken from the build directory, but for MSVC from the project directory?
It may be better to always take it from the project directory so that there is no combinatorics?

Reply:
Good question, next one please... ;-)

Reply:
I guess @anatoly-os might remember the dirty details?